react-cors-spa/src/App.js

```diff
@@ -22,38 +22,50 @@ import logoCF from './logoCloudFront.png';
 import './App.css';
 import axios from 'axios';
 import { useState, useEffect } from 'react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 
 // To be replaced by the endpoint of the API deployed through the CloudFormation Template
-const APIEndPoint = 'to be replaced with your api endpoint here'
+const APIEndPoint = 'https://api.quotable.io/random'
+
+// Create a client
+const queryClient = new QueryClient();
 
 function App() {
   return (
-    <div className="App">
-        <header className="App-header">
-          {APIEndPoint.startsWith('http') &&
-            <APIResult />
-          }
-          <img src={logo} className="App-MainLogo" alt="logo" />
-        </header>
-        <p>This react-based application is hosted in an S3 bucket exposed through a CloudFront distribution</p>
-        <div className="logos">
-            <img src={logoS3} className="App-logoR2L" alt="logo S3" />
-            <img src={logoCF} className="App-logoL2R" alt="logo CloudFront" />
-        </div>
-    </div>
+    <QueryClientProvider client={queryClient}>
+      <div className="App">
+          <header className="App-header">
+            {APIEndPoint.startsWith('http') &&
+              <APIResult />
+            }
+            <img src={logo} className="App-MainLogo" alt="logo" />
+          </header>
+          <p>This react-based application is hosted in an S3 bucket exposed through a CloudFront distribution</p>
+          <div className="logos">
+              <img src={logoS3} className="App-logoR2L" alt="logo S3" />
+              <img src={logoCF} className="App-logoL2R" alt="logo CloudFront" />
+          </div>
+      </div>
+      {/*Floating Mode running locally*/}
+      {process.env.NODE_ENV === 'development' && (
```

Comment:
Oh I'm pretty sure this code won't have access to env vars because it's browser JS not node. I could be wrong though. A better solution is to check the url though.

Reply:
I'll look into this